jake/audit/audit.py

```diff
@@ -16,30 +16,59 @@
 import logging
 
 from typing import List
+from textwrap import wrap
 from colorama import Fore
+from terminaltables import DoubleTable
 
 from ..types.coordinateresults import CoordinateResults
 from ..types.vulnerabilities import Vulnerabilities
 
-class Audit():
+class Audit:
   """ Audit does the business, it prints results from OSS Index to the standard out """
+
   def __init__(self, quiet=False):
-    self._log = logging.getLogger('jake')
+    self._log = logging.getLogger("jake")
     self._quiet = quiet
 
   def audit_results(self, results: List[CoordinateResults]):
     """
     audit_results is the ingest point for the results from OSS Index,
     and handles control flow
     """
-    self._log.debug("Results recieved, %s total results", len(results))
+    self._log.debug("Results received, %s total results", len(results))
 
     total_vulns = 0
     pkg_num = 0
+    good = [x for x in results if len(x.get_vulnerabilities()) == 0]
+    bad = [x for x in results if len(x.get_vulnerabilities()) > 0]
+
+    if len(good) != 0:
+      print()
+      print("Non-Vulnerable Dependencies")
+      print()
+      for coordinate in good:
+        pkg_num += 1
+        total_vulns += self.print_result(coordinate, pkg_num, len(results))
```

Comment:
I do not consider this a blocker, but now that we appear to split out "good" and "bad" results, the logic wherein `self.print_result()` feels a little overly complex and may no longer be needed? Or maybe I'm just misunderstanding how it works, and it is still needed to determine the true total if a result can have more than 1 vulnerability?

Either way, I would not consider adding any change related to the above comment a requirement for this PR, in fact, such changes would probably muddy the water.

Reply:
Yeah this change was a step in the direction you are talking about. I figured I wouldn't bite it off in this PR, but it seems to make sense to do exactly what you are saying!